app/controllers/registration_controller.rb

```diff
@@ -9,7 +10,7 @@ def tracksheet
     name     = params[:name] || 'tracksheet'
     sequence = params[:sequence] || 1
     response['content-disposition'] = "attachment; filename=#{name}-#{sequence}.pdf"
-    pdf = help.generate_tracking_pdf(druids)
+    pdf = generate_tracking_pdf(druids)
```

Comment:
Does this method really belong in a helper, or maybe it should be its own class/model?